libr/bin/bin.c

```diff
@@ -723,11 +740,66 @@ R_API RList *r_bin_classes_from_symbols (RBinFile *bf, RBinObject *o) {
 	return classes;
 }
 
+
+R_API void r_bin_sorted_section_free (void  /*RBinSortedSection*/ *data) {
+	R_FREE(data);
+}
+
+static int r_bin_sorted_section_cmp (const RBinSortedSection *a, const
```

Comment:
single line, in r2land, function definitions doesnt have a space before the ( and function calls to

Reply:
Could you send me clang-format that reflects your style requirements? I've been using the one from radare2//doc/clang-format. Was it right?